tools_dict:
  молоток: ['молоток', 'молот', 'кувалда']
  отвертка: ['отвертка', 'отверточка']
  гаечный ключ: ['гаечный ключ', 'ключ']
  плоскогубцы: ['плоскогубцы', 'пассатижи']
  ножовка: ['ножовка', 'пила']
  рулетка: ['рулетка', 'метр']
  дрель: ['дрель', 'шуруповерт']
  стамеска: ['стамеска']
  уровень: ['уровень']
  нож: ['нож', 'резак']

voice_commands:
  take: ['возьми', 'принеси', 'достань', 'подай']
  put_back: ['положи', 'верни', 'убери']
  stop: ['стоп', 'остановись', 'хватит']
  help: ['помощь', 'команды', 'что ты умеешь']
